_source/_posts/2022-08-08-react-redux.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: blog_post
+title: Using Redux with React
```

Comment:
```suggestion
title: Use Redux to Manage Authenticated State in a React App
```

Scores better on title analyzer. We might play with this a bit more after the article is written